boot/bootutil/src/swap_move.c

```diff
@@ -480,6 +482,9 @@ swap_run(struct boot_loader_state *state, struct boot_status *bs,
 
         if (g_last_idx >= first_trailer_idx) {
             BOOT_LOG_WRN("Not enough free space to run swap upgrade");
+            BOOT_LOG_WRN("required %d bytes but only %d are available",
+                         (g_last_idx + 1) * sector_sz ,
+                         first_trailer_idx * sector_sz);
```

Comment:
```suggestion
            BOOT_LOG_WRN("required %" PRIu32 " bytes but only %" PRIu32 "  are available",
                         (g_last_idx + 1) * sector_sz,
                         first_trailer_idx * sector_sz);
```
`g_last_idx`, `first_trailer_idx` and `sector_sz` are all fixed-width types, also unsigned values. We'd better use the format specifiers from `inttypes.h` to avoid any compiler warnings or printing issues with the underlying implementation.
nit: removed dangling whitespace